tasks/main.yml

```
@@ -34,6 +34,28 @@
   import_tasks: ca.yml
   when: ca_ca_host == inventory_hostname
 
+- name: Set temporary variable ca_san to hostname, FQDN, and inventory hostname
+  set_fact:
+    ca_san: "DNS:{{ ansible_hostname }},DNS:{{ ansible_fqdn }},DNS:{{ inventory_hostname }}"
```

```suggestion
    ca_san: >-
      "DNS:{{ ansible_hostname }},
      DNS:{{ ansible_fqdn }},
      DNS:{{ inventory_hostname }}"
```
You came up with the better solution for a line too long. And since you already strip the string of spaces, we can use it here as well.